src/applications/disability-benefits/686c-674/config/form.js

```diff
@@ -68,15 +68,13 @@ const formConfig = {
   version: migrations.length,
   migrations,
   prefillEnabled: true,
-  footerContent: FormFooter,
-  getHelp: GetFormHelp,
+  verifyRequiredPrefill: true,
   savedFormMessages: {
     notFound: 'Please start over to apply for declare or remove a dependent.',
     noAuth:
       'Please sign in again to continue your application for declare or remove a dependent.',
   },
-  title: 'Add or remove dependents from your VA benefits',
```

Comment:
Not sure how this change got in here, but I think this snippet needs to be reverted.